source/config/__init__.py

```diff
@@ -738,7 +742,7 @@ def renameProfile(self, oldName, newName):
 		if oldName.lower() != newName.lower() and os.path.isfile(newFn):
 			raise ValueError("A profile with the same name already exists: %s" % newName)
 
-		os.rename(oldFn, newFn)
+		osreplace.replace(oldFn, newFn)

```

Comment:
Why the change to osreplace here? In this case, the destination file is not supposed to exist. We should have picked that up above, but it still seems unnecessary... unless I'm missing something?